src/components/molecules/SpaceEditFormNG/SpaceEditFormNG.scss

```diff
@@ -21,6 +21,16 @@ $textarea-height: 120px;
     text-align: center;
   }
 
+  &__input {
+    &-reactionMute {
+      display: flex;
+    }
+
+    &-subtext {
+      margin-right: $spacing--md;
+    }
+  }
+
```

Comment:
Are these even used anywhere? If not please remove, and if so, please do refactor as per above comment.

Reply:
removed ef337b8